### DISCUSSION TITLE
Is there a way to stop wrapping after N lines.

### DISCUSSION THREAD

**Discussion by Kazagan** [QuestPDF/QuestPDF#743] [Q&A] [answered]
Something like the stop paging, where content will be lost, but for maximum wrapping?

**Comment by DawidRobertNowak** [comment, marked as answer]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

This is now supported via [ClampLines](https://www.questpdf.com/api-reference/text/paragraph-style.html#clamp-line-with-ellipsis), which limits a paragraph to a fixed number of lines and discards the overflow. Calling `.Text(content).ClampLines(3)` caps the text at three lines, and you can optionally pass a custom ellipsis as a second argument to mark the truncation.

Should you encounter any further problems, please don't hesitate to open a new issue.
Thank you!
